terraform/deployments/cluster-services/aws_ebs_csi_driver.tf

```diff
@@ -0,0 +1,33 @@
+resource "helm_release" "csi_driver" {
+  chart            = "aws-ebs-csi-driver"
+  name             = "aws-ebs-csi-driver"
+  namespace        = "kube-system"
+  repository       = "https://kubernetes-sigs.github.io/aws-ebs-csi-driver"
+  version          = "1.6.2" # TODO: Dependabot or equivalent so this doesn't get neglected.
+
+  set {
+    name  = "controller.serviceAccount.create"
+    value = true
+  }
+  set {
+    name  = "controller.serviceAccount.name"
+    value = data.terraform_remote_state.cluster_infrastructure.outputs.aws_ebs_csi_driver_controller_service_account_name
+  }
+  set {
+    name  = "enableVolumeResizing"
+    value = true
+  }
+  set {
+    name  = "node.tolerateAllTaints"
+    value = true
```

Comment:
This seems surprising. Why do we want the ebs-plugin pods to tolerate **all** taints? The defaults are `CriticalAddonsOnly` and (for 5m) `NoExecute`, which seem pretty sensible.